### DISCUSSION TITLE
Remove NX

### DISCUSSION THREAD

**Discussion by ComradePashka** [nrwl/nx#29397] [Feature Requests]
Hello!

I am working on web3 project which was ready to deploy about month ago. But project template was using NX I'm still solving bunch of different issues completely not related to my tasks, trying to build/deploy/configure/reset/repair it to finally go live...

So now I only want REMOVE NX FOREVER and use my monorepo on my own.

How can I do that quickly without headache? 

Thank you.

**Comment by jaysoo** [comment]
Closing this issue as it is not constructive. Please read the [Code of Conduct](https://github.com/nrwl/nx/blob/master/CODE_OF_CONDUCT.md) prior to opening issues or discussions.
